ci-scripts/yocto.groovy

```diff
@@ -135,6 +132,13 @@ void runSmokeTests(String yoctoDir, String imageName) {
     }
 }
 
+void runBitbakeTests(String yoctoDir) {
```

Comment:
I just realized that this might never be run, since we don't ever call the function, but I guess it's nice to include it anyway. I think I got it as part of another PR during the merge or something.